code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -166,7 +166,7 @@
 	var/mob/living/M = loc
 
 	if(M.a_intent in accepted_intents)
-		if(istype(M.mind.martial_art, /datum/martial_art/cqc))
```

Comment:
Youve butchered your branch here

Reply:
Yeah I'm not too sure what happened here. According to the more mentally well-equipped coders my VSC was being funky. I'm not trying to sneak in any naughty code.